ui/src/containers/organisations.list/users/invite.tsx

```diff
@@ -0,0 +1,199 @@
+import { zodResolver } from "@hookform/resolvers/zod";
+import { Button, Flex, Sheet, Text } from "@raystack/apsara";
+import { CSSProperties, useCallback, useEffect, useState } from "react";
+import { FormProvider, useForm } from "react-hook-form";
+import { useNavigate, useParams } from "react-router-dom";
+import * as z from "zod";
+import { SheetFooter } from "~/components/sheet/footer";
+import { SheetHeader } from "~/components/sheet/header";
+import { Form } from "@radix-ui/react-form";
+import { CustomFieldName } from "~/components/CustomField";
+import { useFrontier } from "@raystack/frontier/react";
+import { PERMISSIONS } from "~/utils/constants";
+import Skeleton from "react-loading-skeleton";
+import { V1Beta1Group, V1Beta1Role } from "@raystack/frontier";
+import { toast } from "sonner";
+
+const inviteSchema = z.object({
+  type: z
+    .string()
+    .transform((value) => value?.split(",").map((str) => str.trim())),
+  team: z
+    .string()
+    .optional()
+    .transform((value) => value?.split(",").map((str) => str.trim())),
+  emails: z
+    .string()
+    .transform((value) => value.split(",").map((str) => str.trim()))
+    .pipe(z.array(z.string().email())),
+});
+
+type InviteSchemaType = z.infer<typeof inviteSchema>;
+
+export default function InviteUsers() {
+  const { organisationId } = useParams();
+  const { client } = useFrontier();
+  const navigate = useNavigate();
+  const [isDataLoading, setIsDataLoading] = useState(false);
+  const [roles, setRoles] = useState<V1Beta1Role[]>([]);
+  const [groups, setGroups] = useState<V1Beta1Group[]>([]);
+
+  const methods = useForm<InviteSchemaType>({
+    resolver: zodResolver(inviteSchema),
+    defaultValues: {},
+  });
+
+  const onOpenChange = useCallback(() => {
+    navigate(`/organisations/${organisationId}/users`);
+  }, [organisationId]);
+
+  const onSubmit = async (data: InviteSchemaType) => {
+    try {
+      if (!organisationId) return;
+      await client?.frontierServiceCreateOrganizationInvitation(
+        organisationId,
+        {
+          user_ids: data?.emails,
+          group_ids: data?.team,
+          role_ids: data?.type,
+        }
+      );
+      toast.success("members added");
+      navigate(`/organisations/${organisationId}/users`);
+    } catch ({ error }: any) {
+      toast.error("Something went wrong", {
```

Comment:
Can we handle 400s differently here, instead of throwing a generic `Something went wrong`?